pkg/core/nemesis.go

```diff
@@ -105,7 +115,8 @@ func (NoopNemesisGenerator) Generate(nodes []cluster.Node) []*NemesisOperation {
 	ops := make([]*NemesisOperation, len(nodes))
```

Comment:
Does this means the amount of nemesis is equal to amount of nodes?

Reply:
@you06 Sure, you can take a look at the code below.

Reply:
Yes, NoopNemesisGenerator is a faker generator, `NoopNemesis` does nothing actually.